### DISCUSSION TITLE
Error executing vim.schedule lua callback: Vim:E474: Unidentified byte: around:0

### DISCUSSION THREAD

**Discussion by Blairi** [neoclide/coc.nvim#3831] [Q&A] [answered]
How can fix this error?

Error executing vim.schedule lua callback: Vim:E474: Unidentified byte: around:0
stack traceback:
        [C]: in function 'json_decode'
        ...ri/.vim/plugged/completion-nvim/lua/completion/hover.lua:333: in function 'autoOpenHoverInPopup'
        ...e/blairi/.vim/plugged/completion-nvim/lua/completion.lua:196: in function 'cb'
        vim.lua:285: in function <vim.lua:285>

I use default configuration
Thanks! <3

![image](https://user-images.githubusercontent.com/79164448/169575481-c7f4a3f4-e428-452f-add6-1cb0ec1d2238.png)

My nvim version
![image](https://user-images.githubusercontent.com/79164448/169576529-1cd124dd-e20e-4975-bdc0-c7acff8830f2.png)

**Comment by NoahTheDuke** [comment, marked as answer]
That's an error in the plugin completion-nvim, not coc.nvim.

**Comment by chemzqm** [comment]
Seems it tried to parse `user_data` of complete item as json data, which could be wrong.
